tests/pytorch/test_dataloader.py

```diff
@@ -195,6 +236,100 @@ def test_neighbor_sampler_dataloader():
         assert isinstance(iter(dl), Iterator)
         _check_neighbor_sampling_dataloader(_g, nid, dl, mode, collator)
 
+@unittest.skipIf(F._default_context_str == 'gpu', reason="GPU sample neighbors not implemented")
+def test_etype_neighbor_sampler_dataloader():
```

Comment:
how does this test `sample_etype_neighbors`?

Reply:
This test case should be deleted as we will not support etype neighbor sample for no-DistGraph.